packages/py/polywrap-core/.gitignore

```diff
@@ -157,4 +157,6 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
\ No newline at end of file
+#.idea/
+
+.packages/py/__pycache__
```

Comment:
I think that inside of `packages/py`, we should create a `.gitignore` file and add `__pycache__`. Lastly, removing this change

Reply:
done